bin/collator/src/parachain/service.rs

```diff
@@ -60,16 +60,6 @@ pub mod astar {
     /// Shibuya runtime executor.
```

Comment:
Since you're updating the `local/services.rs`, might be also good to fix this comment (change it to Astar).